src/main/java/com/chopshop166/chopshoplib/sensors/PigeonGyro.java

```diff
@@ -0,0 +1,44 @@
+package com.chopshop166.chopshoplib.sensors;
+
+import com.ctre.phoenix.sensors.PigeonIMU;
+
+import edu.wpi.first.wpilibj.GyroBase;
+
+public class PigeonGyro extends GyroBase {
+
+    final private PigeonIMU gyro;
+
+    public PigeonGyro(final PigeonIMU gyro) {
```

Comment:
We should also provide a constructor that will create the PigeonIMU object from a [TalonSRX](http://www.ctr-electronics.com/downloads/api/java/html/classcom_1_1ctre_1_1phoenix_1_1sensors_1_1_pigeon_i_m_u.html#a2e6fa81264b980c02b70619f7134848e)